Correct format for the table [Limits on entities]

elbruno opened the pull request:
There was a couple of invalid spaces which makes the table to be displayed as regular text

azurecla commented:
Hi __@elfisher__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

elbruno committed:
Correct format for the table [Limits on entities]

There was a couple of invalid spaces which makes the table to be displayed as regular text
